docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,63 @@
+# Release 458 (13 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause. ({issue}`23087`)
+* Improve error message while dereferencing an unknown row field. ({issue}`23370`)
```

Comment:
We don’t need to document improvements to error messages.

Reply:
nixed